### ISSUE TITLE
Nhận xét bài làm

### ISSUE THREAD

**Issue by tonyvu1289** [tonyvu1289/P4DS-FinalProject-vip#3]
Các bạn nhận xét đồ án nhóm mình nhẹ tay nhé. :-)

**Comment by tonyvu1289** [comment]
test

**Comment by Taka304** [comment]
h

**Comment by MinhVuong2000** [comment]
Theo như mô tả của dữ liệu, có 615 dòng và 14 cột, tuy nhiên khi bạn gõ `loan_df.shape.shape` chỉ có `(500,14)`. Các bạn có thể lý giải chỗ này tại sao không?

**Comment by MinhVuong2000** [comment]
Đối với dữ liệu outlier, đôi khi nó làm nhiễu, nhưng đôi khi nhờ vào đó, bạn cũng có thể tìm ra insight rất hay, đặc biệt là trong lĩnh vực kinh tế tài chính. Ví dụ như có một khách hàng cực kì giàu, thu nhập cực khủng, vượt xa so với những người còn lại, họ chính là tiềm năng đem đến nguồn lợi lớn cho doanh nghiệp; Hay ví dụ như các trường hợp gian lận tín dụng, chúng thường là đều là các outlier. Do đó việc bỏ đi outlier mà chưa xem xét chúng có ý nghĩa gì hay không thì khá là uổng data. Do đó, nếu được, các bạn có thể xem xét thêm về ý nghĩa của các dòng outlier mà các bạn đã drop để xem chúng có mang đến insight gì hay ho cho các bạn không :D

**Comment by tonyvu1289** [comment]
> Cám ơn chị đã nhận xét. Em xin lý giải như sau : 
- Có lẽ phần mô tả của dữ không chính xác, vì khi bọn em download dataset về kiểm tra bằng mắt thì đúng là chỉ có 500 dòng thật.
- Một trường hợp khác là do dataset đúng là có 615 dòng nhưng đã được bên phía upload xử trước trước rồi nên còn có 500 dòng thôi.

**Comment by tonyvu1289** [comment]
> 

Vâng, bọn em sẽ xem xét lại xử lý outlier. Có lẽ kinh nghiệm là thứ bọn em thiếu trong trường hợp này :(( .
